### PR TITLE
chore: update node lts and update stylelint

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] tests are included
-   [x] documentation is changed or added
-   [x] [message properties][] have been updated with new phrases
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

Updates node to latest long term support release.

Update stylelint to latest version, and resolves new lint errors detected.

This should not change how the build runs, or how the page is styled.
It focuses on improving code styles and having additional long term support

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: uPortal-webapp/src/main/webapp/media/skins/respondr/common/common.less [hunk 1]
```diff
@@ -30,32 +30,32 @@
  * NOTE:  Do not change the location of Bootstrap sources lightly;  they must
  * be included in the uPortal.war package.
  */
-@import "../../../../webjars/bootstrap/less/bootstrap.less";
+@import url("../../../../webjars/bootstrap/less/bootstrap.less");
```
Comment (ChristianMurphy) [inline uPortal-webapp/src/main/webapp/media/skins/respondr/common/common.less:33]: these imports appear to resolve exactly the same, but make it clearer that it is a browser path

File: uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/gallery.less [hunk 2]
```diff
@@ -958,20 +955,15 @@
...
-  border-top-right-radius: 0;
   -moz-border-radius-topright: 0;
-  -webkit-border-top-right-radius: 0;
+  border-top-right-radius: 0;
   padding: 0;
   margin: 0;
   list-style: none;
-  -moz-box-shadow: 0 0 4px 0 #333;
-  -webkit-box-shadow: 0 0 4px 0 #333;
   box-shadow: 0 0 4px 0 #333;
```
Comment (ChristianMurphy) [inline uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/gallery.less:963]: these got folded together, no browser needs these prefixed anymore.
https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

File: uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less [hunk 3]
```diff
@@ -306,7 +266,7 @@
                 min-width: 100%;
                 color: @black;
 
-                &:lastchild {
+                &:last-child {
```
Comment (ChristianMurphy) [inline uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less:269]: stylelint caught this typo

File: uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less [hunk 4]
```diff
@@ -457,6 +403,7 @@
             }
           }
         }
+
```
Comment (ChristianMurphy) [inline uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less:406]: stylelint adds a newline between blocks of css

**Review by bjagg — APPROVED** [review]
Awesome! Some great changes here 🥇

**Pull request merged**